packages/login-with-qq/template/home_security.html

```diff
@@ -0,0 +1,211 @@
+{% extends "layout/home_base.html" %}
```

Comment:
Template replacement is NOT allowed in official plugin.

Reply:
If I need to add elements at the end of the page, how do I do it.

I'm sorry, I haven't done the front-end before. I'm a bit unfamiliar with the things here. Please give me more guidance.